### DISCUSSION TITLE
CANNOT Send or Receive Emails After New Installation

### DISCUSSION THREAD

**Discussion by aaronesteban1** [modoboa/modoboa#2035] [Q&A]
# Impacted versions

* Modoboa: latest version
* installer used: command line
* Webserver: Nginx/Apache

I have downloaded the application through the command line, have gone through the basic installation process, have made sure that port 25 is open, have enabled ports 25, 80, 443 through UFW, and done everything I thought that I could do to make it work.

Here is the error log located in "/var/log/mail.log"
`
Oct 15 01:04:39 mail postfix/postscreen[14824]: CONNECT from [209.85.222.44]:34857 to [10.10.0.5]:25
Oct 15 01:04:39 mail postfix/postscreen[14824]: WHITELISTED [209.85.222.44]:34857
Oct 15 01:04:39 mail postfix/smtpd[14825]: connect from mail-ua1-f44.google.com[209.85.222.44]
Oct 15 01:04:39 mail postfix/smtpd[14825]: Anonymous TLS connection established from mail-ua1-f44.google.com[209.85.222.44]: TLSv1.3 with cipher TLS_>
Oct 15 01:04:39 mail postfix/smtpd[14825]: NOQUEUE: client=mail-ua1-f44.google.com[209.85.222.44]
Oct 15 01:04:39 mail postfix/smtpd[14825]: warning: connect to proxy filter inet:[127.0.0.1]:10024: Connection refused
Oct 15 01:04:39 mail postfix/smtpd[14825]: proxy-reject: END-OF-MESSAGE: 451 4.3.0 Error: queue file write error; from=<nekajbeauty@gmail.com> to=<ad>
Oct 15 01:04:39 mail postfix/cleanup[14831]: DADFFC1457: message-id=<20201015010439.DADFFC1457@mail.aaronsnewsletters.com>
Oct 15 01:04:39 mail postfix/smtpd[14825]: disconnect from mail-ua1-f44.google.com[209.85.222.44] ehlo=2 starttls=1 mail=1 rcpt=1 bdat=0/1 quit=1 com>
Oct 15 01:04:39 mail postfix/qmgr[11786]: DADFFC1457: from=<double-bounce@mail.aaronsnewsletters.com>, size=1461, nrcpt=1 (queue active)
Oct 15 01:04:39 mail postfix/local[14832]: DADFFC1457: to=<root@mail.aaronsnewsletters.com>, orig_to=<postmaster>, relay=local, delay=0.01, delays=0.>
Oct 15 01:04:39 mail postfix/qmgr[11786]: DADFFC1457: removed
Oct 15 01:05:31 mail dovecot: imap-login: Login: user=<user1@aaronsnewsletters.com>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=15169, secured,>
Oct 15 01:05:37 mail postfix/postscreen[14824]: CONNECT from [209.85.218.68]:36806 to [10.10.0.5]:25
Oct 15 01:05:37 mail postfix/postscreen[14824]: WHITELISTED [209.85.218.68]:36806
Oct 15 01:05:37 mail postfix/smtpd[14825]: connect from mail-ej1-f68.google.com[209.85.218.68]
Oct 15 01:05:37 mail postfix/smtpd[14825]: Anonymous TLS connection established from mail-ej1-f68.google.com[209.85.218.68]: TLSv1.3 with cipher TLS_>
Oct 15 01:05:37 mail postfix/smtpd[14825]: NOQUEUE: client=mail-ej1-f68.google.com[209.85.218.68]
Oct 15 01:05:37 mail postfix/smtpd[14825]: warning: connect to proxy filter inet:[127.0.0.1]:10024: Connection refused
Oct 15 01:05:37 mail postfix/smtpd[14825]: proxy-reject: END-OF-MESSAGE: 451 4.3.0 Error: queue file write error; from=<aaronestebanmarketing@gmail.c>
Oct 15 01:05:37 mail postfix/cleanup[14831]: F2E41C1457: message-id=<20201015010537.F2E41C1457@mail.aaronsnewsletters.com>
Oct 15 01:05:38 mail postfix/smtpd[14825]: disconnect from mail-ej1-f68.google.com[209.85.218.68] ehlo=2 starttls=1 mail=1 rcpt=1 bdat=0/1 quit=1 com>
Oct 15 01:05:38 mail postfix/qmgr[11786]: F2E41C1457: from=<double-bounce@mail.aaronsnewsletters.com>, size=1471, nrcpt=1 (queue active)
Oct 15 01:05:38 mail postfix/local[14832]: F2E41C1457: to=<root@mail.aaronsnewsletters.com>, orig_to=<postmaster>, relay=local, delay=0.02, delays=0.>
Oct 15 01:05:38 mail postfix/qmgr[11786]: F2E41C1457: removed
Oct 15 01:07:56 mail postfix/anvil[13588]: statistics: max connection rate 1/60s for (smtpd:209.85.208.182) at Oct 15 00:57:56
Oct 15 01:07:56 mail postfix/anvil[13588]: statistics: max connection count 1 for (smtpd:209.85.208.182) at Oct 15 00:57:56
Oct 15 01:07:56 mail postfix/anvil[13588]: statistics: max cache size 2 at Oct 15 01:04:39
Oct 15 01:13:31 mail dovecot: imap-login: Login: user=<user1@aaronsnewsletters.com>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=16476, secured,>
`

Also, I've logged in as a user that was created by the main admin account. I've tried sending from admin and user1. I've tried to send from my gmail account as well. Nothing is working (sending or receiving email).

Any support is appreciated.

**Comment by MrGeneration** [comment]
The message that should raise your awareness is:
`Oct 15 01:04:39 mail postfix/smtpd[14825]: warning: connect to proxy filter inet:[127.0.0.1]:10024: Connection refused`

By default this would be amavis not running for whatever reason.
`systemctl start amavis` should solve this issue. 

Edit: By the way: This is not modoboa specific in case you wondered. :)

**Comment by aaronesteban1** [comment]
**I have already tried to restart/reload both Postfix & amavis to no avail.
Please see the latest send and receive error log that I've attached here.**

![Modoboa-Cant-Send-or-receive](https://user-images.githubusercontent.com/58304956/96075982-4b79bb00-0e7a-11eb-9aff-dbb08035d63a.PNG)

**Comment by tonioo** [comment]
Looks like amavis refuses to start. Can you check why?

**Comment by aaronesteban1** [comment]
> Looks like amavis refuses to start. Can you check why?

Hi Tonioo,

I'm not 100% sure how to check it. I'm new to setting up mail servers on Linux Ubuntu. I'm actually more into VB.NET, PHP, Javascript (HTML & CSS of course). I actually do more software development.

Maybe it's when I was doing the installation. I noticed that there were options to change some passwords for MySQL and another application, but I don't remember for sure since the installation was so quick.

I could always just reinstall everything starting over from scratch. It was such a quick set up that it doesn't really affect anything since I don't have any data anyhow. I was just curious to see if we could fix it prior to redoing everything.

But just in case this happens again, you guys seem to believe that the best thing to check is "amavis" I see.

Also, after running the setup, completing the installation process, and creating a new user from the admin section, are there any other command lines to run or any additional work to start sending and receiving or is it supposed to work straight out of the box after that? Thanks again.

**Comment by MrGeneration** [comment]
If a service isn't starting or crashing during startup, you can quickly check those situations with `systemctl status amavis` (you can replace amavis by any other service running on your system). This will provide the last 5 lines or so and *usually* contains the information you're looking for.

Also, if you start a service manually and it crashed, your system will tell you as soon as the prompt returns.

---

You can find a installation how to within the documentation:
https://modoboa.readthedocs.io/en/latest/installation.html

It really contains everything you need to get the job done.

**Comment by aaronesteban1** [comment]
I just did a new installation of Modoboa and had everything working. After installing Mautic on top of this Modoboa, I am no longer able to receive any email for any of the email accounts that I've created in Modoboa. What do you think that I should try to do to fix this? Any ideas on what could be stopping the messages from coming in? I'm still able to send email through Modoboa, but not able to receive after the Mautic installation. Thanks again Marcel.

**Comment by MrGeneration** [comment]
I don't think Mautic has anything to do with that issue as it's a PHP website basically.
Your log file (mail.log) should contain hints on where to look.

If you still have connection refused errors, you'll need to solve those.
